public/js/dom.js

```diff
@@ -0,0 +1,44 @@
+// const url = `https://api.themoviedb.org/3/search/movie?api_key=${config.apiKey }&language=en-US&query=`;
+function selecElement(id) {
```

Comment:
i think this function is not useful   
cause you will repeat ``` selectElement ``` each time you need to select item